src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -293,6 +296,26 @@ fun processMacroSimpleResolveVariants(element: RsMacroBodySimpleMatching, proces
     return simple.any { processor(it) }
 }
 
+fun processMetaItemResolveVariants(element: RsMetaItem, processor: RsResolveProcessor): Boolean {
+    if (element.parentOfType<RsMetaItem>()?.identifier?.text != "derive") return false
+    val traitName = element.identifier.text
+    val stdTrait = STD_DERIVE_TRAITS[traitName]
```

Comment:
No, if `traitName` is not in `STD_DERIVE_TRAITS` it means that it's some other name, for example, `Serialize` from some library. In this case, I consider that it's trait name and I try to find it but if there are at least two different traits with the same name I can find the wrong declaration. If you consider that it isn't ok, I'll resolve only derivable traits from stdlib.